Framework/SINQ/src/PoldiPeakSearch.cpp

```diff
@@ -47,7 +47,8 @@ PoldiPeakSearch::PoldiPeakSearch()
   * @param correlationCounts :: Vector with correlation counts.
   * @return Vector with sum of neighboring correlation counts.
   */
-MantidVec PoldiPeakSearch::getNeighborSums(MantidVec correlationCounts) const {
+MantidVec
+PoldiPeakSearch::getNeighborSums(const MantidVec &correlationCounts) const {
```

Comment:
Is the work for this class going to be covered in another issue? It seems to have leftover `MantidVec` in places